Jenkinsfile

```diff
@@ -26,7 +26,7 @@ def job = {
     }
 
     if (config.publish && config.isDevJob) {
-      configFileProvider([configFile(fileId: 'Gradle Nexus Settings', variable: 'GRADLE_NEXUS_SETTINGS')]) {
+      configFileProvider([configFile(fileId: 'Gradle-Artifactory-Settings', variable: 'GRADLE_NEXUS_SETTINGS')]) {
```

Comment:
Do we need to rename `GRADLE_NEXUS_SETTINGS`?

Reply:
i think we can leave it as is for now, i believe the variable is being used in downstream scripts and configs.